python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -145,7 +145,10 @@ def __init__(self, *, config=None):
             self.outputs.remove("inputMap")
 
         if not self.config.doWriteArtifactMasks:
-            self.outputs.remove("artifactMasks")
+            try:
+                self.outputs.remove("artifactMasks")
+            except KeyError:
+                pass
```

Comment:
We can't just catch the error and pass. We should work on this in pair coding tomorrow.